lib/charms/opensearch/v0/opensearch_distro.py

```diff
@@ -127,6 +127,10 @@ def stop(self):
         while self.is_started() and (datetime.now() - start).seconds < 60:
             time.sleep(3)
 
+    def get_service_status(self):
```

Comment:
Do we need this? we already have:
- `is_started`
- `is_failed`
- `is_node_up`